### ISSUE TITLE
Execute all tests found within the tests directory

### ISSUE THREAD

**Issue by vhartikainen** [gpestana/kapacitor-unit#26]
It would be helpful to provide just the tests directory on command line. Then the tool would iterate over all tests it finds and runs the tests. 

e.g. 
```
scripts
+- scripts1
     +- script1.tick
+- scripts2
     +- script2.tick
tests
+- test1.yaml
+- test2.yaml
```

And running:
`kapacitor-unit --dir scripts --tests tests/`

The tool would then execute all tests it finds from the directory

**Comment by gpestana** [comment]
Yep, this sounds like a good improvement! Would you push the ball forward @vhartikainen ? 😃 

The cli already parses all the tick scripts in a directory when input in `--dir script`. Maybe that code can be reused?

**Comment by vhartikainen** [comment]
A big maybe from me 😏  Never written a single line of Go, but I'm willing to give it a go. 

No promises on estimate when to be finished though 😄

**Comment by vlaurenzano** [comment]
Pr #33 accomplishes this. It checks the --tests config to see if it's a dir or a yaml file before parsing so it's backwards compatible.
